runtime/doc/vim_diff.txt

```diff
@@ -662,6 +662,7 @@ These legacy Vim features are not yet implemented:
 - *'guitabtooltip'* *'gtt'*
 - *'mouseshape'* *'mouses'*
 - *'previewpopup'*
+- *popupwin*. You can use |api-floatwin| instead.
```

Comment:
I doubt we will implement this interface, so it can go in the "Editor" section of `nvim-removed`.